doc/configuration-flow.md

```diff
@@ -0,0 +1,54 @@
+# Overview of the configuration flow in Istio Manager
+
+Istio configuration consists of configuration objects. Each object has kind
+(for example, a routing rule), name and namespace uniquely identifying the
+object.  Istio Manager provides a REST-based interface for manipulating objects
```

Comment:
This makes it sound like I can  do
`curl https://istio_manager/...  -X PUT -d "{}"`, if that is not the case this comment is misleading. 
If it *is* the case, we should show `curl` examples.